spec.html

```diff
@@ -25279,7 +25279,7 @@ <h1>Object ( [ _value_ ] )</h1>
         <emu-alg>
           1. If NewTarget is neither *undefined* nor the active function, then
             1. Return ? OrdinaryCreateFromConstructor(NewTarget, *"%Object.prototype%"*).
-          1. If _value_ is *null*, *undefined* or not supplied, return ObjectCreate(%Object.prototype%).
+          1. If _value_ is *undefined* or *null*, return ObjectCreate(%ObjectPrototype%).
```

Comment:
I think you accidentally changed `%Object.prototype%`.

Reply:
Thank you, fixed in 0afecec.